web/about.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>About YOU.</title>
+
+<meta name="viewport" content="width=device-width">
+<meta name=”robots” content="index, follow">
+<meta name="author" content="Anonymous">
+<meta itemprop="name" content="About You.">
+<meta itemprop="description" content="About You. -- The privacy-first period-tracking app">
+<meta itemprop="image" content="https://youperiod.app/images/logo-social-small.png">
+<meta name="twitter:card" content="summary">
+<meta name="twitter:title" content="About You.">
+<meta name="twitter:description" content="About You. -- The privacy-first period-tracking app">
+<meta name="twitter:image" content="https://youperiod.app/images/logo-social-small.png">
+<meta name="twitter:image:alt" content="You. App">
+<meta name="title" property="og:title" content="About You.">
+<meta name="description" property="og:description" content="About You. -- The privacy-first period-tracking app">
+<meta name="image" property="og:image" content="https://youperiod.app/images/logo-social-large.png">
+<meta name="url" property="og:url" content="https://youperiod.app/">
+<meta name="site_name" property="og:site_name" content="You.">
+<meta name="type" property="og:type" content="website">
+
+<link rel="icon" type="image/x-icon" href="/favicon.ico" />
+<link rel="stylesheet" href="css/style.css">
+<script integrity="sha256-CoCYJ/tTxH9vJyISOUlowiGKF8OokDL5QBuS3H8R1/g=">
+// compute the vw/vh units more reliably than CSS does itself
+if (document.documentElement && document.documentElement.style && document.documentElement.style.setProperty) {
+	let docEl = document.documentElement;
+	let width = Math.max(300,docEl.clientWidth);
+	let height = Math.max(300,docEl.clientHeight);
+	docEl.style.setProperty("--vw-unit",`${(width / 100).toFixed(1)}px`);
+	docEl.style.setProperty("--vh-unit",`${(height / 100).toFixed(1)}px`);
+}
+</script>
+</head>
+<body>
+
+<header>
+
+<h1 id="logo">YOU<i class="bold-period" aria-label="stylized period in red color"></i></h1>
+
+<nav>
+	<ul>
+		<li><a href="/">Home</a></li>
+		<li><a href="about.html">About</a></li>
+		<li><a href="privacy-policy.html">Privacy Policy</a></li>
```

Comment:
I don't think this PR includes this "privacy-policy" file, so let's leave this out of the nav and instead have each page's PR add its own nav entry.

Same with the nav markup in index.